<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_face</class>
 <widget class="QDialog" name="T3_AF_face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="_timeLabel_">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>50</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>_timeLabel_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_dateLabel_">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>30</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>_dateLabel_</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_exitPushBtn_">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>8</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>_exitPushBtn_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_faceGifLabel_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>160</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>_faceGifLabel_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_textLabel_">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>_textLabel_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_squareLabel_">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>105</y>
     <width>535</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/face_square.png)</string>
   </property>
   <property name="text">
    <string>_squareLabel_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_videoLabel_">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>106</y>
     <width>357</width>
     <height>268</height>
    </rect>
   </property>
   <property name="text">
    <string>_videoLabel_</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="_logBackgroundLabel_">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>106</y>
     <width>177</width>
     <height>226</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/face_logBackground.png)</string>
   </property>
   <property name="text">
    <string>_logBackgroundLabel_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_enterFaceLogPushBtn_">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>65</y>
     <width>123</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/face_button.png);color:white</string>
   </property>
   <property name="text">
    <string>人脸信息管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_morePushBtn_">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>105</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_vocalPushBtn_">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>27</y>
     <width>123</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/face_button.png);color:white</string>
   </property>
   <property name="text">
    <string>语音编辑</string>
   </property>
  </widget>
  <widget class="QListView" name="_logListView_">
   <property name="geometry">
    <rect>
     <x>588</x>
     <y>140</y>
     <width>175</width>
     <height>189</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
  </widget>
  <widget class="QLabel" name="_headLabel_">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>20</y>
     <width>112</width>
     <height>67</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/face_head.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_battery_">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="_battIMG_">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>_timeLabel_</zorder>
  <zorder>_dateLabel_</zorder>
  <zorder>_exitPushBtn_</zorder>
  <zorder>_faceGifLabel_</zorder>
  <zorder>_textLabel_</zorder>
  <zorder>_logBackgroundLabel_</zorder>
  <zorder>_videoLabel_</zorder>
  <zorder>_enterFaceLogPushBtn_</zorder>
  <zorder>_vocalPushBtn_</zorder>
  <zorder>_squareLabel_</zorder>
  <zorder>_morePushBtn_</zorder>
  <zorder>_logListView_</zorder>
  <zorder>_headLabel_</zorder>
  <zorder>_battery_</zorder>
  <zorder>_battIMG_</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
